djstripe/event_handlers.py

```diff
@@ -238,16 +211,18 @@ def _handle_crud_like_event(
 
 	verb = verb or event.verb
 	customer = customer or event.customer
-	crud_type = crud_type or CrudType.determine(event=event, verb=verb, exact=crud_exact)
 	obj = None
 
-	if crud_valid and not crud_type.valid:
+	try:
+		crud_type = crud_type or CrudType.determine(event=event, verb=verb)
+	except ValueError:
+		# Ignore events like .succeeded or .paid
```

Comment:
I'm not super familiar with this bit of code, but I think this logic is wrong - currently we do a sync on  .succeeded , .paid etc events.

Should we treat these as UPDATED events instead?

Reply:
probably, but we'd need a strict mapping, and possibly an "OTHER" enum: https://stripe.com/docs/api/events/types